cram_common/cram_object_knowledge/src/household.lisp

```diff
@@ -325,3 +325,440 @@
+                         (owl-name "kitchen_sink_block_counter_top")
+                         (part-of ?kitchen-name)))
+           (side left)
+           (side front)
+           (range-invert 0.5)))
+
+(defmethod man-int:get-object-likely-location :heuristics 20
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-setting))
+     (?object-type (eql :spoon)))
+  
+  (desig:a location
+           (in (desig:an object
+                         (type drawer)
+                         (urdf-name sink-area-left-upper-drawer-main)
+                         (owl-name "drawer_sinkblock_upper_open")
+                         (part-of ?kitchen-name)))
+           (side front)))
+
+(defmethod man-int:get-object-likely-location :heuristics 20
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-setting))
+     (?object-type (eql :knife)))
+  
+  (desig:a location
+           (in (desig:an object
+                         (type drawer)
+                         (urdf-name sink-area-left-upper-drawer-main)
+                         (owl-name "drawer_sinkblock_upper_open")
+                         (part-of ?kitchen-name)))
+           (side front)))
+
+(defmethod man-int:get-object-likely-location :heuristics 20
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-setting))
+     (?object-type (eql :plate)))
+
+  (desig:a location
+           (on (desig:an object
+                         (type counter-top)
+                         (urdf-name sink-area-surface)
+                         (owl-name "kitchen_sink_block_counter_top")
+                         (part-of ?kitchen-name)))
+           (side left)
+           (side front)
+           (range-invert 0.5)))
+
+(defmethod man-int:get-object-likely-location :heuristics 20
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-setting))
+     (?object-type (eql :mug)))
+
+  (desig:a location
+           (side left)
+           (on (desig:an object
+                         (type counter-top)
+                         (urdf-name sink-area-surface)
+                         (owl-name "kitchen_sink_block_counter_top")
+                         (part-of ?kitchen-name)))))
+
+(defmethod man-int:get-object-likely-location :heuristics 20
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-setting))
+     (?object-type (eql :cup)))
+  
+  (desig:a location
+           (side left)
+           (on (desig:an object
+                         (type counter-top)
+                         (urdf-name sink-area-surface)
+                         (owl-name "kitchen_sink_block_counter_top")
+                         (part-of ?kitchen-name)))))
+
+(defmethod man-int:get-object-likely-location :heuristics 20
+    ((?kitchen-name (eql :kitchen))
+     ?human-name 
+     (?context (eql :table-setting))
+     (?object-type (eql :breakfast-cereal)))
+
+  (desig:a location
```

Comment:
should switch to left vertical drawer according to milestone scenario. however, test beforehand if opening the drawer is possible. should be.